meta-dts-distro/recipes-bsp/txe-secure-boot/txesbmantool_git.bb

```diff
@@ -0,0 +1,39 @@
+DESCRIPTION = "Utility for generating TXE Secure Boot manifests"
+LICENSE = "GPL-2.0-only"
+
+SRC_URI = " \
+git://github.com/Dasharo/coreboot.git;branch=txe_sb_tool;protocol=https \
+"
+
+SRCREV = "295c158e577400025d01e7ca8ad3beb64e1614a0"
+
+LIC_FILES_CHKSUM = " \
+file://${WORKDIR}/git/LICENSES/GPL-2.0-only.txt;md5=5430828348d2cf7d4b5e8395f774a68e \
+"
+
+S = "${WORKDIR}/git/util/txesbmantool"
+
+inherit pkgconfig
+
+DEPENDS += "wolfssl"
+
+do_configure:prepend(){
+    export PKG_CONFIG_PATH="${STAGING_LIBDIR}/pkgconfig:${STAGING_DIR_HOST}/usr/lib/pkgconfig"
+}
+
+do_compile:prepend() {
+    export PKG_CONFIG_PATH="${STAGING_LIBDIR}/pkgconfig:${STAGING_DIR_HOST}/usr/lib/pkgconfig"
+}
+
+do_install() {
+    oe_runmake 'DESTDIR=${D}' install
+}
+
+INSANE_SKIP:${PN} += "ldflags"
+
+prefix = "/usr/local"
```

Comment:
defintely a bad idea. The smmstoretool PR comments apply here

Reply:
Done 28e45f9ffe9e01a4b1e5d45d933ed98f16ff668b